src/search/search.component.html

```diff
@@ -6,8 +6,11 @@
 		'bx--search--xl': size === 'xl',
 		'bx--search--light': theme === 'light',
 		'bx--skeleton': skeleton,
-		'bx--toolbar-search': toolbar,
-		'bx--toolbar-search--active': toolbar && active
+		'bx--toolbar-search': toolbar && !tableSearch,
+		'bx--toolbar-search--active': toolbar && !tableSearch && active,
+		'bx--toolbar-search-container-persistent': tableSearch && !toolbar,
+		'bx--toolbar-search-container-expandable': tableSearch && toolbar,
```

Comment:
Should this be
```suggestion
		'bx--toolbar-search-container-expandable': tableSearch && toolbar && expandable,
```
otherwise expandable input has no effect

Reply:
so expandable is not defined in the Search component, and the previous implementation was setting toolbar property to trigger the expandable in the table. (same answer for the comment below)